workflow/environments/README.md

```diff
@@ -49,24 +60,24 @@ pip install XXX --upgrade
 - Failing to activating the environemt and running `pip install` will end up with the packages installed under `$HOME/.local/lib`. Double-check if this directory has anything installed. Anything installed there can get in the way, and loaded instead of the one installed in the environment.
```

Comment:
```suggestion
- Failing to activating the environment and running `pip install` will end up with the packages installed under `$HOME/.local/lib`. Double-check if this directory has anything installed. Anything installed there can get in the way, and loaded instead of the one installed in the environment.
```